Samples/CustomOperator/desktop/cpp/custom-operator-sample.vcxproj

```diff
@@ -9,7 +9,7 @@
     <ProjectGuid>{2bf804d4-daa2-42be-9f21-0e94f021ef53}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>SqueezeNetObjectDetection</RootNamespace>
-    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>

```

Comment:
Can you verify that the custom operator sample still works for registering new operators why building against 19H1 SDK? I believe that the functionality may depend on a RS5 SDK version currently, in which case the code should be modified.

Reply:
Such functionality failure will be caught by Sample Tests pipeline. And the pipeline will tell us whether code should be modified. What do you think?

---
In reply to: [409809598](https://github.com/microsoft/Windows-Machine-Learning/pull/310#discussion_r409809598) [](ancestors = 409809598)

Reply:
I believe the pipeline currently will just skip that part of the custom op code if the agent isn't running RS5. So if you are going to rely on the samples pipeline then, just make sure it runs on an RS5 agent, but with a 19H1 SDK. 

Ideally we should figure out a way to make the sample version independent if it is an issue.

Reply:
I think the dependency might just be at runtime though, so perhaps the SDK doesn't matter.